shared/chat/inbox/row/container.js

```diff
@@ -122,7 +122,10 @@ const ConnectedRow = pausableConnect(
     return (state: TypedState) => selector(state)
   },
   dispatch => ({
-    onSelectConversation: (key: ConversationIDKey) => dispatch(selectConversation(key, true)),
+    onSelectConversation: (key: ConversationIDKey) => {
+      dispatch(setInboxFilter(''))
+      dispatch(selectConversation(key, true))
```

Comment:
This is a bit jarring since if the selected conversation is outside of the visible rows with the filter removed. Not sure if that's worth fixing...